ml-agents/mlagents/trainers/torch/model_serialization.py

```diff
@@ -40,6 +40,36 @@ def is_exporting():
         return exporting_to_onnx._local_data._is_exporting
 
 
+class TensorNames:
+    BatchSizePlaceholder = "batch_size"
```

Comment:
Please follow python naming conventions for these. I understand that you probably wanted to copy-paste the C# code here, but it makes the resulting python code stick out.

Reply:
Ok, will do. 
Just to make sure, are we okay with this approach of having a class with static strings or is there a more Pythonic way to do this?

Reply:
You could do it as a enum too (python enums aren't constrained to integer values) - see `BufferKey` in buffer.py, but there's not much advantage to doing that here (and you'd have to write a bit more code to cast to strings).

You could also add methods to the class for combining the prefix + index, like `TensorNames.get_visual_observation_name(i)`

Reply:
I wanted to have as much as possible the same code between C# and Python for the TensorNames, but I think adding helper methods would help.

Reply:
> I think adding helper methods would help.

It's right there in the name :)

Fee free to add to C# too.

Reply:
done